test-cases/longevity/longevity-counters-multidc.yaml

```diff
@@ -24,3 +24,6 @@ internode_encryption: 'dc'
 ip_ssh_connections: 'public'
 
 use_legacy_cluster_init: false
+
+# Temporarily downgrade scylla_bench to a stable version
+scylla_bench_version: v0.1.3
```

Comment:
I don't think the issue reproduced here, so please check (in the spreadsheet) and if not remove it from here.

Reply:
One of the s-b issues was reproduced here - https://github.com/scylladb/scylla-bench/issues/89